src/index.js

```diff
@@ -0,0 +1,17 @@
+import 'core-js/stable'
```

Comment:
I don’t think this is needed anymore [since we use `corejs: 3` in the preset-env config](https://github.com/zloirock/core-js#babelruntime)?

Reply:
Only when using the runtime afaik, I think `preset-env` still needs this as an entry point before transform.

Reply:
Oh you’re right, this is needed because we are using `useBuiltIns: 'entry'`. Should we move to `useBuiltIns: 'usage'`? :thinking: https://babeljs.io/docs/en/babel-preset-env#usebuiltins-usage

Reply:
Missed that opt, nice!